relativedeltafield/tests/test_relativedeltafield.py

```diff
@@ -186,7 +176,6 @@ def test_timedelta_input(self):
 		# Derived value, from the number of days (see #2)
 		self.assertStrictEqual(1, obj.value.weeks)
 
-
 	def test_filtering_works(self):
 		obj1 = Interval(value='P1Y3M1W4.5DT5H70.5M80.10001S')
 		obj1.save()
```

Comment:
Also in this file there are a lot of unnecessary whitespace changes.

I'd appreciate it if you could also add actual tests for the form field.